src/panels/config/ha-panel-config.ts

```diff
@@ -82,67 +230,45 @@ class HaPanelConfig extends LitElement {
     if (isComponentLoaded(this.hass, "cloud")) {
       this._updateCloudStatus();
     }
+    this.style.setProperty(
```

Comment:
Why can't we put this in the CSS?

Reply:
Now it applies to all config pages, and this is a router element